### PR TITLE
DEV-2196 Revengine Coverage

### PR DESCRIPTION
**Pull request by njharman**
#### What's this PR do?
Add test coverage, only 4 files < 95% coverage. Almost got to 98%

- Add revengine to coverage report.
- Cut revengine/storages.py unused, untested.
- Increase revengine/* coverage to 100%.
- Increase apps/* coverage to get total coverage > 98%
- Cut things that were untested, broken, and unused.
- Do not ignore testing simple things like __str__(). Fixed several errors.

#### Why are we doing this? How does it help us?
Less bugs

#### How should this be manually tested? Please include detailed step-by-step instructions.
N/A

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
No

#### Have automated unit tests been added? If not, why?
Indeed

#### How should this change be communicated to end users?
N/A

#### Are there any smells or added technical debt to note?
There's a bunch of pragma no-cover on code that should probably be tested.

Had weird issues with ApplePayDomain, missing RevenueProgram._get_host().

#### Has this been documented? If so, where?
N/A

#### What are the relevant tickets? Add a link to any relevant ones.
[DEV-2196]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
No

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
Discovered maybe tests are making network connections [DEV-2244]

[DEV-2196]: https://news-revenue-hub.atlassian.net/browse/DEV-2196?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: revengine/tests/test_celery.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from revengine.celery import debug_task
+
+
+def test_debug_task():
```
Comment (x110dc) [inline revengine/tests/test_celery.py:4]: What's this testing? Maybe I'm misunderstanding.

File: revengine/urls.py [hunk 2]
```diff
@@ -45,11 +45,15 @@
...
+    path(
+        ".well-known/apple-developer-merchantid-domain-association",
+        read_apple_developer_merchant_id,
+        name="apple_dev_merchantid_domain",
+    ),
 ]
 
 
-if settings.DEBUG:
+if settings.DEBUG:  # prgma: no cover Is covered, but reimport confuses coverage
```
Comment (x110dc) [inline revengine/urls.py:56]: pragma misspelling

**Review by x110dc — CHANGES_REQUESTED** [review]
One question and one minor change request.

File: revengine/tests/test_celery.py (hunk 1 above)
Comment (njharman) [reply revengine/tests/test_celery.py:4]: The debug_task, obviously! :)

I don't know why debug_task() exists, but it wasn't covered. It consists only of a print statement. This just tests function runs without error. Mostly just "covers" the code.

**Review by x110dc — APPROVED** [review]

**Pull request merged**